lib/webpack-config/create-webpack-config.test.js

```diff
@@ -46,6 +46,70 @@ test('Basic Test', () => {
   ).resolves.toMatchSnapshot();
 });
 
+test('Adds mapbox-gl to exclude if user is on 2.0.0 or above', () => {
+  getUserConfig.mockResolvedValueOnce({});
+  const tempDir = tempy.directory();
+  fs.writeFileSync(
+    path.join(tempDir, 'package.json'),
+    `{
+      "name": "fake",
+      "version": "0.0.0",
+      "dependencies": {
+        "mapbox-gl": "^2.0.0"
+      }
+    }`
+  );
+
+  fs.writeFileSync(
+    path.join(tempDir, 'babel.config.js'),
+    `module.exports = {}`
+  );
+
+  const urcPromise = config(
+    getCliOpts({
+      configPath: path.join(tempDir, 'underreact.config.js')
+    })
+  );
+
+  return urcPromise.then(urc => {
+    const config = createWebpackConfig(urc);
+    expect(config.module.rules[0].oneOf[1].include).toEqual(
+      /\/node_modules\/((?!mapbox-gl\/))/
+    );
+  });
+});
+
+test('Does not add mapbox-gl to exclude if user is on a version below 2.0.0', () => {
+  getUserConfig.mockResolvedValueOnce({});
+  const tempDir = tempy.directory();
+  fs.writeFileSync(
+    path.join(tempDir, 'package.json'),
+    `{
+      "name": "fake",
+      "version": "0.0.0",
+      "dependencies": {
+        "mapbox-gl": "~1.9.0"
+      }
+    }`
+  );
+
+  fs.writeFileSync(
+    path.join(tempDir, 'babel.config.js'),
+    `module.exports = {}`
+  );
+
+  const urcPromise = config(
+    getCliOpts({
+      configPath: path.join(tempDir, 'underreact.config.js')
+    })
+  );
+
+  return urcPromise.then(urc => {
+    const config = createWebpackConfig(urc);
+    expect(config.module.rules[0].oneOf[1].include).toBeUndefined();
+  });
+});
```

Comment:
Could use a couple more tests:
- Error case if user provides both include and exclude lists
- Case where use provides their own set of exclude modules